portal/system_uri.py

```diff
@@ -13,7 +13,9 @@
 # Auth identities - typically used with suffix for provider (i.e. google)
 # or to name identity type (i.e. TrueNTH-username)
 TRUENTH_IDENTITY_SYSTEM = '{}/identity-codes'.format(TRUENTH_NAMESPACE)
-TRUENTH_ID ='{system}/{provider}'.format(
+TRUENTH_EXTERNAL_STUDY_SYSTEM = '{system}/external-study-id'.format(
+    system=TRUENTH_IDENTITY_SYSTEM)
```

Comment:
We're simply using that as a namespace.  See other FHIR systems as examples.  I don't see any need and expect it'd only create bugs to distinguish on ePROMs/TrueNTH.